Automattic/jetpack
# JS Components: Convert PricingCard component to TypeScript

#### Changes proposed in this Pull Request:
* This PR converts `PricingCard` component to TypeScript and cleans up the propTypes.
* It also converts the `PricingCard` stories and unit tests to TS

#### Other information:


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Checkout the PR
* Run `cd projects/js-packages/storybook && pnpm run storybook:dev`
* Confirm that all the stories of `PricingCard` component work as expected
* Check the consumers of `PricingCard` and related components and confirm that they work as expected
* Check for any regression


<img width="709" alt="Screenshot 2022-06-30 at 3 55 20 PM" src="https://user-images.githubusercontent.com/18226415/176655142-c0031825-e3f0-453d-8088-bf6257fb82f2.png">



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202276314263989
